enterprise/tpa_pipeline.py

```diff
@@ -56,6 +68,13 @@ def handle_enterprise_logistration(backend, user, **kwargs):
 
     """
     request = backend.strategy.request
+    # Social user account is new or already binded with edx account
+    new_association = kwargs.get('new_association', True)
+
+    if is_multiple_user_enterprises_feature_enabled() and not new_association:
+        if backend.name in get_oauth2_backend_names():
```

Comment:
I don't think we need these checks. This code is going to be executed while SSO pipeline is running